src/pages/workspace/WorkspaceInvitePage.tsx

```diff
@@ -187,7 +187,9 @@ function WorkspaceInvitePage({route, policy}: WorkspaceInvitePageProps) {
         // Filtering out selected users from the search results
         const selectedLogins = selectedOptions.map(({login}) => login);
         const personalDetailsWithoutSelected = Object.values(personalDetails).filter(({login}) => !selectedLogins.some((selectedLogin) => selectedLogin === login));
-        const personalDetailsFormatted = personalDetailsWithoutSelected.map((item) => OptionsListUtils.formatMemberForList(item));
+        const personalDetailsFormatted = personalDetailsWithoutSelected
+            .map((item) => OptionsListUtils.formatMemberForList(item))
+            .filter((item, index, array) => item?.login && index === array.findIndex((i) => i.login === item.login));
```

Comment:
> We should move the `filter` logic before `map`.

@dukenv0307 Any specific reason for this request ? Like performance, proneness to error or anything else, curious to hear why you think the way it is now needs to be changed.

> Should we check item?.login? We already filterout the item that doesn't have login [here](https://github.com/Expensify/App/blob/653af04f8bf51661eb18f1c48a56c1362e5c61b2/src/libs/OptionsListUtils.ts#L1187-L1189), so I think we don't need that check.

That filters out entries with **non-existing** `login` (and other) key values, assuming you're familiar with the issue you just reviewed, the fix is filtering out **two entries with the same** `login` key value since we don't want to show duplicates in case BE returns such entries.